packages/deployed-backend-client/src/stack_metadata_output_retrieval_strategy.ts

```diff
@@ -66,13 +66,13 @@ export class StackMetadataBackendOutputRetrievalStrategy
           (tag) => tag.Key === 'amplify:deployment-type'
         )?.Value ?? 'sandbox';
       throw new BackendOutputClientError(
-        BackendOutputClientErrorType.METADATA_RETRIEVAL_ERROR,
+        BackendOutputClientErrorType.DEPLOYMENT_IN_PROGRESS,
         `This ${deploymentType} deployment is in progress. Re-run this command once the deployment completes.`
       );
     }
     if (outputs === undefined) {
       throw new BackendOutputClientError(
-        BackendOutputClientErrorType.METADATA_RETRIEVAL_ERROR,
+        BackendOutputClientErrorType.NO_OUTPUTS_FOUND,
```

Comment:
@johnpc fyi I'm proposing this change to better articulate the backend output client error, so that at the cli client side we can infer it easily from the error type. Let me know if this change is safe for console.